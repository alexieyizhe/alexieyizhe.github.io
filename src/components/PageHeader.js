import React from "react";
import styled from "styled-components";

const Header = styled.div`
  display: inline-block;
  margin-bottom: 5vmax;
  font-size: 6vh;
  font-weight: bold;
`;

const PageHeader = (props) => (
  <Header>{props.title}</Header>
);

export default PageHeader;
