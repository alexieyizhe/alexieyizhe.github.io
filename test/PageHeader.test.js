import React from 'react';
import PageHeader from '../src/components/PageHeader';

describe('Page Header', () => {

  it('should render the header', () => {
    const header = render(
      <PageHeader className="pageHeader" title="Test Header" />
    );

    expect(header).toMatchSnapshot();
  });
});
