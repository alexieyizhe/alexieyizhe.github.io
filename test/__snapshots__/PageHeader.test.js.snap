// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Header should render the header 1`] = `
<div
  class="sc-bdVaJa jUbhmH"
>
  Test Header
</div>
`;
